deps/rabbit/src/rabbit_quorum_queue.erl

```diff
@@ -2186,12 +2186,13 @@ leader_health_check(QueueNameOrRegEx, VHost, ProcessLimitThreshold) ->
 
 run_leader_health_check(ClusterName, QResource, HealthCheckRef, From) ->
     Leader = ra_leaderboard:lookup_leader(ClusterName),
-    case ra_server_proc:ping(Leader, ?LEADER_HEALTH_CHECK_TIMEOUT) of
-        {pong,leader} ->
-            From ! {ok, HealthCheckRef, QResource};
-        _ ->
-            From ! {error, HealthCheckRef, QResource}
-    end,
+    {_, _, _} =
```

Comment:
Sounds like `_ = …` then would do?

Reply:
yeah, dialyzer didnt complain for both. switching to `_` if thats preferred 👍